src/components/executive-education-2u/ExecutiveEducation2UPage.jsx

```diff
@@ -30,13 +33,41 @@ function ExecutiveEducation2UPage() {
     isExecEd2UFulfillmentEnabled,
   });
 
+  const createExecutiveEducationFailureMessages = (failureCode) => {
+    const failureCodeMessages = {
+      no_offer_available: 'No offer is available to cover this course.',
+      no_offer_with_enough_balance: 'Your organization doesn’t have sufficient balance to cover this course.',
```

Comment:
nit: we could move the logic dealing with failure messaging to its own component.